src/libs/conduit/conduit_node.cpp

```diff
@@ -4385,7 +4393,8 @@ Node::set_external_data_using_schema(const Schema &schema,
 {
     reset();
     m_schema->set(schema);
-    walk_schema(this,m_schema,data);
+#warning "should set_external_data_using_schema inherit allocator?"
```

Comment:
What needs to be done here?

Reply:
i do think it should inherit -- external vs alloced are orthogonal paths